examples/go/go.mod

```diff
@@ -1,4 +1,4 @@
-module twilio-oai-generator/go
+module github.com/twilio/twilio-go
```

Comment:
Won't this be a problem since the published module is called exactly that?

Reply:
This is actually what we want, which is the terraform-provider thinking this is the real twilio-go repo since it depends on it.